### ISSUE TITLE
Update current tables with caption component for accessibility reasons

### ISSUE THREAD

**Issue by RLHecht** [department-of-veterans-affairs/va.gov-team#9717]
## User story
As a user of the website with a screenreader, I need the caption component of tables to help me better understand the information encompassed in the table.

As a content team member I need to ensure that all pages on VA.gov are accessible to all. 

## Issue Description
When someone is using a screenreader on a page with tables, if important information about the table is placed in the content block above it, the screenreader doesn't read it as part of the table. Therefore, we need to use the caption component of the table content block to identify what the table is about. 

In rates tables we need to identify that the rates are listed in U.S. dollars. This should be put in the caption component instead of in the WYSIWIG above. See screenshot for example:![Screen Shot 2020-06-02 at 12.05.37 PM.png](https://images.zenhubusercontent.com/5c9b7e302ac54336e2c327b8/5b590af8-0b14-494a-aaac-d512319a2e1e)

---
## Tasks
Identify and update all live pages that currently contain tables and add them as a checklist here to keep track of. I've started a list but there are definitely more:

- [ ] https://www.va.gov/disability/compensation-rates/veteran-rates/
- [ ] https://www.va.gov/disability/compensation-rates/special-benefit-allowance-rates/
- [ ] https://www.va.gov/disability/compensation-rates/birth-defect-rates/
- [ ] https://www.va.gov/disability/survivor-dic-rates/
- [ ] https://www.va.gov/disability/parent-dic-rates/
- [ ] https://www.va.gov/pension/veterans-pension-rates/
- [ ] https://www.va.gov/pension/survivors-pension-rates/

## Acceptance Criteria
- [ ] All tables on live site contain a caption header
---
## How to configure this issue
- [ ] **Attached to a Milestone** (when will this be completed?)
- [ ] **Attached to an Epic** (what body of work is this a part of?)
- [ ] **Labeled with Team** (`product support`, `analytics-insights`, `operations`, `service-design`, `tools-be`, `tools-fe`)
- [ ] **Labeled with Practice Area** (`backend`, `frontend`, `devops`, `design`, `research`, `product`, `ia`, `qa`, `analytics`, `contact center`, `research`, `accessibility`, `content`)
- [ ] **Labeled with Type** (`bug`, `request`, `discovery`, `documentation`, etc.)

**Comment by caw310** [comment]
The va-table component does add the table title to a caption component for the table. 
If teams migrate to the new va-table web component the caption component will automatically contain the tables title.

**Comment by caw310** [comment]
@DanielleThierryUSDSVA , do you need assistance with contacting teams to update the table component to the va-table web component?

**Comment by DanielleThierryUSDSVA** [comment]
Hi @caw310. This is specific to Drupal pages, and was an issue that I had flagged for the content team a long while back. Our previous content team did some work around this already, so I'm wondering if this is just an old ticket. We're in between content teams right now so I'll re-assign this to the new team once they're onboarded and ask them to continue to look into this. No action needed for now. Thanks!

**Comment by laflannery** [comment]
@DanielleThierryUSDSVA I am currently going through backlog issues and came across this.  Would you be able to tell me if

- this is something for the sitewide content team (going through and adding captions within Drupal) or
- should it be public websites because they need to switch these tables to the proper design system component?

Thanks!

**Comment by DanielleThierryUSDSVA** [comment]
Hi @laflannery. I'm going to bring @RLHecht in here for an update on this one. Thanks!
